package.json

```diff
@@ -7,6 +7,8 @@
   "typings": "dist/index.d.ts",
   "scripts": {
     "build": "tsdx build",
+    "extract": "api-extractor run --local --verbose",
+    "generate": "api-documenter markdown --input-folder ./extraction --output-folder ./docs/reference",
```

Comment:
How do you feel about naming these commands a little more clearly? Maybe something like `docs:extract` and `docs:generate`, with a `docs:build` that runs both of those serially?